### PR TITLE
kernel/binary_manager : Fix build warning

### PR DESCRIPTION
**Pull request by YiHyunJin**
1. If it is not initialized, it is initialized.
2. The unused variables were deleted.
3. The return type was modified.

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [1f9bb763232e07218c24444bcbc9ba283cdacb39] - Code Rule Check Result:
os/kernel/binary_manager/binary_manager_callback.c:270: ERROR: [SPC_M_KWD] space required before the open parenthesis '('

**Comment by seinfra** [comment]
Target : [1f9bb763232e07218c24444bcbc9ba283cdacb39] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [a3053c493f74f19f6cd6c1209ba0b4e26a000b83] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [a3053c493f74f19f6cd6c1209ba0b4e26a000b83] - Code Rule Check Result:
os/kernel/binary_manager/binary_manager_callback.c:270: ERROR: [SPC_M_KWD] space required before the open parenthesis '('

File: os/kernel/binary_manager/binary_manager_callback.c [hunk 1]
```diff
@@ -189,7 +189,7 @@ int binary_manager_send_statecb_msg(int recv_binidx, char *bin_name, uint8_t sta
 	int ret;
 	int send_count;
 	mqd_t send_mq;
-	mqd_t recv_mq;
+	mqd_t recv_mq = NULL;
```
Comment (sunghan-chang) [inline os/kernel/binary_manager/binary_manager_callback.c]: I think you had better modify recv_mq logic itself without this modification.

File: os/kernel/binary_manager/binary_manager_recovery.c [hunk 2]
```diff
@@ -128,13 +124,13 @@ static int recovery_exclude_scheduling(int binid)
  *   None
  *
  ****************************************************************************/
+#ifdef CONFIG_BINMGR_RECOVERY
```
Comment (sunghan-chang) [inline os/kernel/binary_manager/binary_manager_recovery.c]: Unnecessary. This file is built when this config is enabled.

**Comment by seinfra** [comment]
Target : [66eea6546be8880474d380f613a1b824c4150f25] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [66eea6546be8880474d380f613a1b824c4150f25] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [08a546e0c86fedea3d142ea33c4d3ffff4e221d5] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [08a546e0c86fedea3d142ea33c4d3ffff4e221d5] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [ed0199d904b5815a5841c54d1f5a58480af304e1] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [ed0199d904b5815a5841c54d1f5a58480af304e1] - Code Rule Check OK.

File: os/kernel/binary_manager/binary_manager_callback.c (hunk 1 above)
Comment (YiHyunJin) [reply os/kernel/binary_manager/binary_manager_callback.c]: It is hard to modify recv_mq  logic.
So it has to initialize.

**Pull request merged**